src/patternfly/components/Alert/alert.scss

```
@@ -3,215 +3,171 @@
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-alert--grid-template-columns: max-content 1fr max-content;
-  --pf-c-alert--grid-template-rows: 1fr auto;
+  --pf-c-alert--grid-column-gap: var(--pf-global--spacer--xs);
```

renamed the var and bumped the value - the current gap is a little tight.

```suggestion
  --pf-c-alert--GridColumnGap: var(--pf-global--spacer--sm);
```